bukkit/src/main/java/dev/aurelium/auraskills/bukkit/requirement/blocks/BlockRequirement.java

```diff
@@ -0,0 +1,36 @@
+package dev.aurelium.auraskills.bukkit.requirement.blocks;
+
+import org.bukkit.Material;
+
+import java.util.LinkedList;
+
+public class BlockRequirement {
+
+    private final Material material;
+    private final boolean checksPlace;
+    private final boolean checksBreak;
+    private final LinkedList<RequirementNode> nodes;
```

Comment:
Use the abstract List type here instead of LinkedList, same for all other places in this class.